src/sentry/spans/consumers/process/flusher.py

```diff
@@ -118,41 +119,50 @@ def produce(payload: KafkaPayload) -> None:
                     producer_futures.append(producer.produce(topic, payload))
 
             while not stopped.value:
-                now = int(time.time()) + current_drift.value
-                flushed_segments = buffer.flush_segments(max_segments=max_flush_segments, now=now)
+                with metrics.timer("spans.buffer.flusher.loop_body"):
```

Comment:
This measurement includes the `time.sleep` below which seems unfortunate, since we're just idling - it doesn't tell us how fast the loop can actually process data. Otherwise, all the relevant measurements are already covered by `flusher.produce` below. I'd opt to keep only the produce timings below, but not the entire loop.

Reply:
done